implementations/rust/ockam/ockam_api/src/cli_state/cli_state.rs

```diff
@@ -43,7 +46,9 @@ pub struct CliState {
     database: SqlxDatabase,
     application_database: SqlxDatabase,
     tracing_enabled: TracingEnabled,
-    pub(crate) progress_bar_reporting_channel_sender: Sender<ReportingChannelMessageType>,
+    /// Broadcast channel to be notified of major events during a process supported by the
+    /// CliState API
+    notifications: Sender<Notification>,
```

Comment:
I like the name / concept of notifications rather than progress message reporting 👍